src/js/components/presentational/styled/Buttonify.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Button = styled.button`
+    width: ${({width}) => (width ? `${width}px` : '50px')};
```

Comment:
Lets switch the pixels to say 1 or 2rem (based on whatever looks good).

Reply:
Makes sense, updating now